applications/portal/frontend/src/components/Home/AntibodiesTable.tsx

```diff
@@ -124,9 +129,16 @@ const RenderCellContent = (props: GridRenderCellParams<String>) => {
       component="div"
       className="col-content"
     >
-      {props.field === "targetAntigen"
-        ? `${props.row.abTarget} ${props.row.targetSpecies.join(", ")}`
-        : props.value}
+      {
+        props.field === "ab_target" ? `${props.row.abTarget} ${props.row.targetSpecies.join(", ")}` :
```

Comment:
An elegant refactoring here would be the using a lookup table of functions like:

```javascript
const values_map = {
  ab_target: (row) => `${props.row.abTarget} ${props.row.targetSpecies.join(", ")}`
 ...
}
```

then 

```
{
 values_map[props.field]
}

```